### PR TITLE
Sgomena/hotfix

### PR DESCRIPTION
**Pull request by samgomena**

### CODE REVIEW

**Review by lixphilosophy — DISMISSED** [review]

**Review by fennnn — DISMISSED** [review]

File: src/build/settings/base.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "app_name": "Pluto",
+    "author": "DeepMarket",
+    "main_module": "src/main/python/main.py",
+    "version": "0.0.0"
```
Comment (logyball) [inline src/build/settings/base.json]: this is pretty nitpicky but should we be making this a 1.0.0 or something? 0.0.0 seems to not be giving us enough credit.

File: src/build/settings/linux.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "categories": "Utility;",
+    "description": "",
```
Comment (logyball) [inline src/build/settings/linux.json]: should we add stuff here?  Does this ever get picked up by FBS?  I looked on https://build-system.fman.io/manual/#directory-structure and it doesnt look like this is used by anything but maybe you know something I dont

File: src/main/python/interfaces/util.py [hunk 3]
```diff
@@ -21,8 +21,8 @@ def load_path(path, _file):
     """
     for p in path.split(os.pathsep):
         candidate = os.path.join(p, _file)
-    if os.path.exists(candidate) and os.path.isfile(candidate):
-        return candidate
+        if os.path.exists(candidate) and os.path.isfile(candidate):
```
Comment (logyball) [inline src/main/python/interfaces/util.py:24]: good catch on this here

File: src/main/python/main.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    @cached_property
+    def settings_style(self):
+        return self.load_style("settings.qss")
+
+    @cached_property
+    def question_style(self):
+        return self.load_style("question.qss")
+
+    # def __init__(self):
+    #     popup_style = self.get_resource("popup.qss")
```
Comment (logyball) [inline src/main/python/main.py]: should we just nuke this code? commented out just looks kind of ugly

File: src/main/python/main.py [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+            file_path = self.get_resource(path)
+            with open(file_path) as f:
+                return f.read()
+        except:
+            raise
+
+if __name__ == '__main__':
+    from sys import exit, argv
+
+    # check we're in correct directory
```
Comment (logyball) [inline src/main/python/main.py]: same here

File: src/unittest/python/test_api.py [hunk 6]
```diff
@@ -20,42 +21,39 @@ def server_is_up():
         else:
             # Quasi check for server status
             return all([bool(txt) for txt in ['api', 'is', 'online'] if txt in str(status_request)])
```
Comment (logyball) [inline src/unittest/python/test_api.py:23]: this is something for next round of fixes, but we shouldn't be calling out to the API in any of our unit tests.  I'll work on that when making things local

**Review by logyball — DISMISSED** [review]
added a few comments but nothing serious, mostly nitpicky questions.  Looks like the tests pass and the application builds correctly

File: src/build/settings/base.json (hunk 1 above)
Comment (samgomena) [reply src/build/settings/base.json]: Good call. Updated it to the version displayed in the application.

File: src/build/settings/linux.json (hunk 2 above)
Comment (samgomena) [reply src/build/settings/linux.json]: I'm not entirely sure about how fbs uses these files but I added some placeholders for now.

File: src/main/python/main.py (hunk 4 above)
Comment (samgomena) [reply src/main/python/main.py]: Yah, good call. I removed it.

**Pull request merged**